theme/static/js/resourceLandingAjax.js

```diff
@@ -281,11 +387,21 @@ function share_resource_ajax_submit(form_id) {
 
                 // Form actions
                 var unshareUrl;
-                if (shareType == "user"){
-                    unshareUrl = $form.attr('action').replace("share-resource-with-user", "unshare-resource-with-user") + share_with + "/";
+                var undoUrl;
+                if (shareType == "user") {
+                    unshareUrl =
+                        $form.attr('action').replace("share-resource-with-user", "unshare-resource-with-user")
+                        + share_with + "/";
+
+                    undoUrl = rowTemplate.find(".undo-share-form").attr("action") + share_with + "/";
                 }
                 else {
-                    unshareUrl = $form.attr('action').replace("share-resource-with-group", "unshare-resource-with-group") + share_with + "/";
+                    unshareUrl =
+                        $form.attr('action').replace("share-resource-with-group", "unshare-resource-with-group")
+                        + share_with + "/";
+
+                    undoUrl = rowTemplate.find(".undo-share-form").attr("action").replace("undo-share-resource-with-user", "undo-share-resource-with-group")
```

Comment:
@mauriel133 ```replace``` call must have a typo, since two strings are exactly the same. The first string to be replaced should be ```share-resource-with-user``` I believe.